components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -196,6 +199,7 @@ export default class CollectionItem extends mixins(
       type: '',
       sortBy: (this.$route.query.sort as string) ?? 'BLOCK_NUMBER_DESC',
       listed: false,
+      owned: false,
```

Comment:
default false, I'll never see my nft unless I click on "Own".
it would be better to leave the choice to the user (in a select/option)

Reply:
IF setting default true, users who don't have nft will see nothing in a collection page. 

Seem that the default `false` is more suitable for most people.

Do you prefer to add an option, to set the default value,  on the user setting page?

Reply:
I would make it default `null` or empty

Reply:
ok

Reply:
✅ Tested